utility/src/main/java/org/oppia/android/util/parser/html/CustomBulletSpan.kt

```diff
@@ -91,6 +25,12 @@ class CustomBulletSpan(context: Context) : LeadingMarginSpan {
       spannableStringBuilder: SpannableStringBuilder,
       context: Context
     ): SpannableStringBuilder {
+      val resources = context.resources
+      val bulletRadius = resources.getDimensionPixelSize(R.dimen.bullet_radius)
+      val gapWidth = resources.getDimensionPixelSize(R.dimen.bullet_gap_width)
+      /** The space between the start of the line and the bullet. */
```

Comment:
```suggestion
      // The space between the start of the line and the bullet.
```

This shouldn't be a KDoc.